src/main/java/org/terasology/segmentedpaths/controllers/PathFollowerSystem.java

```diff
@@ -125,6 +170,13 @@ public boolean isVehicleValid(EntityRef vehicleEntity) {
         return true;
     }
 
+    /**
+     * Moves the vehicle for delta distance along the path
+     * @param vehicleEntity Vehicle to be moved
+     * @param delta Distance the vehicle is to be moved by
```

Comment:
If you're aligning param descriptions in other places, do it here too. Stay consistent